Process.c

```diff
@@ -54,7 +54,7 @@ void Process_setupColumnWidths() {
       return;
    }
 
-   Process_pidDigits = ceil(log10(maxPid));
+   Process_pidDigits = (int)log10(maxPid) + 1;
```

Comment:
This may depend on the active rounding mode. Use explicit `floor`

Reply:
@BenBE  No, that's unnecessary. C standard says that the float to int cast always truncate.
Reference: http://www.open-std.org/jtc1/sc22/WG14/www/docs/n1256.pdf
Section 6.3.1.4 "Real floating and integer": _When a finite value of real floating type is converted to an integer type other than `_Bool`, the fractional part is discarded (i.e., the value is truncated toward zero)._